superset-frontend/spec/javascripts/explore/controlUtils_spec.jsx

```diff
@@ -107,6 +109,7 @@ describe('controlUtils', () => {
                   name: 'all_columns',
                   config: {
                     type: 'SelectControl',
+                    controlGroup: 'columns',
```

Comment:
Naming suggestions are welcomed. I went between `requestGroup` and `controlGroup`, bud decided in favor of `controlGroup`, as I expect it will be more intuitive for viz developers.